bindings/python/src/normalizers.rs

```diff
@@ -634,13 +669,23 @@ impl Serialize for PyNormalizerWrapper {
     }
 }
 
-#[derive(Debug, Clone, Deserialize)]
-#[serde(untagged)]
+#[derive(Debug, Clone)]
 pub(crate) enum PyNormalizerTypeWrapper {
     Sequence(Vec<Arc<RwLock<PyNormalizerWrapper>>>),
     Single(Arc<RwLock<PyNormalizerWrapper>>),
 }
 
+impl<'de> Deserialize<'de> for PyNormalizerTypeWrapper {
```

Comment:
can you add a comment as to why we need this?

Reply:
Will do yes